babel.config.extensions.js

```diff
@@ -0,0 +1,58 @@
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				useBuiltIns: 'entry',
+				corejs: 2,
+				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+				exclude: [ 'transform-typeof-symbol' ],
+			},
+		],
+		'@babel/react',
+	],
+	plugins: [
+		'@babel/plugin-proposal-class-properties',
+		'@babel/plugin-proposal-export-default-from',
+		'@babel/plugin-proposal-export-namespace-from',
+		'@babel/plugin-syntax-dynamic-import',
+		[
+			'@babel/transform-runtime',
+			{
+				corejs: false, // we polyfill so we don't need core-js
+				helpers: true,
+				regenerator: false,
+				useESModules: false,
+			},
+		],
+	],
+	overrides: [
+		{
+			test: './src',
```

Comment:
Hmm, this goes back to this code's Calypso days. No `src/` directory in `extensions/`. We might just merge the override into the top level `plugins`.

Reply:
875f5f2